src/streaming/controllers/StreamController.js

```diff
@@ -135,6 +144,68 @@ function StreamController() {
         eventBus.on(Events.MANIFEST_UPDATED, onManifestUpdated, this);
         eventBus.on(Events.STREAM_BUFFERING_COMPLETED, onStreamBufferingCompleted, this);
         eventBus.on(MediaPlayerEvents.METRIC_ADDED, onMetricAdded, this);
+        if (skipDataLoss === true) {
```

Comment:
It would be nice supporting the possibility of enabling/disabling skip gaps feature after player has been initialized. Requires a bit more control from our side (create/destroy listener in setSkipGap).